### DISCUSSION TITLE
Update Angular plugin

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Lern/Community#80] [Query]
As Angular Migration for SunbirdEd . Team we are removing some of the dependency please remove the dependency of the same in your repo.

1.ngx-infinite-scroll
please refer the ED ticket [ED-1276](https://project-sunbird.atlassian.net/browse/ED-1276)
cc: @Krishnaj20 @vinukumar-vs

**Comment by rhwarrier** [comment]
@reshmi-nair  @Hari-stackroute   pls note.

**Comment by vinukumar-vs** [comment]
@reshmi-nair @rhwarrier Can you please share the tentative timeline why when these changes can be release. This is the blocker for Sunbrid-ED to upgrade to Angular-15. 

We have only 2 options.
Option 1: Wait for the the updated NPM library compatible with Angular-15. SunbirdEd Angular-15 upgrade also will be blocked
Option 2: Remove the Discussion-UI module & proceed. This will be having huge impact on adopters who are using discussion-UI.

@Krishnaj20 @chitranshu-keshav

**Reply by reshmi-nair** [reply]
@vinukumar-vs We will check and get back on this. We have only [Discussion Forum-UI](https://lern.sunbird.org/learn/product-and-developer-guide/discussion-forum) that needs to be upgraded.

**Comment by rhwarrier** [comment]
As discussed over email, both the DF and Group pieces need to be addressed. The Lern team will be able to take it up only by 5.4 (post June) as of now.
